edit/fix.go

```diff
@@ -396,6 +375,16 @@ func cleanUnusedLoads(f *build.File) bool {
 			all = append(all, load)
 		} else {
 			fixed = true
+			// If the load statement contains a post-comment, keep it
+			if len(load.Comment().After) == 0 {
```

Comment:
Should we do the same for Comment().Before?

Reply:
I'm not sure, there are indeed cases when a before-comment belongs only to one statement and to all of them. The fact that load statements are not separated by empty lines (unless they have comments) and that there's a (non-default) fix that orders them lexicographically makes it hard to determine which is the case:

    # Bar definitions:
    load(":bar.bzl", "foo")
    load(":foo.bzl", "bar")

vs

    # All definitions:
    load(":bar.bzl", "foo")
    load(":foo.bzl", "bar")

Reply:
Changed to keeping all comments (except suffix).